src/components/JobLogsContainer/JobLogsContainer.js

```diff
@@ -0,0 +1,81 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+
+import {
+  useJobLogsProperties,
+  useJobLogsListFormatter,
+} from '@folio/stripes-data-transfer-components';
+import {
+  Button,
+  Callout,
+} from '@folio/stripes/components';
+import { listTemplate } from '../ListTemplate';
+import sharedCss from '../../shared.css';
+
+import { DEFAULT_JOB_LOG_COLUMNS } from '../../utils';
```

Comment:
Put an empty line between imports from folio, own components imports, and styles imports, please.
See this wiki page for more details - https://wiki.folio.org/pages/resumedraft.action?draftId=14458784&draftShareId=483fa2fb-615e-4002-a916-c976bd5fb8c6&

```suggestion
} from '@folio/stripes/components';

import { listTemplate } from '../ListTemplate';
import { DEFAULT_JOB_LOG_COLUMNS } from '../../utils';

import sharedCss from '../../shared.css';
```

Reply:
done